src/main/scala/uk/gov/nationalarchives/tdr/api/service/TransferAgreementService.scala

```diff
@@ -74,15 +74,19 @@ class TransferAgreementService(consignmentMetadataRepository: ConsignmentMetadat
       transferAgreementCompliance <- consignmentMetadataRepository.addConsignmentMetadata(convertTAComplianceInputToPropertyRows(input, userId)).map(
         rows => convertDbRowsToTransferAgreementCompliance(input.consignmentId, rows)
       )
-      _ <- addTransferAgreementStatus(input.consignmentId, "Completed")
+      _ <- updateTransferAgreementStatus(input.consignmentId, "Completed")
     } yield transferAgreementCompliance
   }
 
-  def addTransferAgreementStatus(consignmentId: UUID, statusValue: String): Future[ConsignmentstatusRow] = {
+  def addTransferAgreementStatus(consignmentId: UUID, statusValue: String="InProgress"): Future[ConsignmentstatusRow] = {
```

Comment:
Yes, this is temporary, my plan is (Once we've deleted the old `addTransferAgreement` method on line 23), to remove the default parameter add `InProgress` as an argument to. `ConsignmentstatusRow` on line 82, i.e. replace `statusValue` (on line 82) with `InProgress` because I came to the conclusion that when you add a TA status, the only status it could/would/should be is an `InProgress` status (as that's the initial status value); everything else will be added via the `updateExistingTransferAgreementStatus`